charts/rstudio/templates/deployment.yml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: r-studio
   labels:
     app: r-studio
```

Comment:
`app` should also use the template fullname, eg `app: {{ template "fullname" . }}`